yjit/bindgen/src/main.rs

```diff
@@ -196,6 +198,7 @@ fn main() {
         .allowlist_function("rb_set_cfp_(pc|sp)")
         .allowlist_function("rb_cfp_get_iseq")
         .allowlist_function("rb_yjit_multi_ractor_p")
```

Comment:
There's a function named `rb_multi_ractor_p`, can we also not export that one directly?